### PR TITLE
[build-script] When building packages install libIndexStore from clang

### PR DESCRIPTION
**Pull request by akyrtzi**
This is needed for the LSP service.

### CODE REVIEW

**Comment by akyrtzi** [comment]
@swift-ci build linux toolchain

**Comment by akyrtzi** [comment]
@swift-ci build toolchain

**Review by shahmishal — APPROVED** [review]
LGTM!

File: utils/build-presets.ini [hunk 1]
```diff
@@ -1062,7 +1062,7 @@ test-installable-package
 reconfigure
 
 swift-install-components=compiler;clang-builtin-headers;stdlib;swift-syntax;sdk-overlay;license;sourcekit-xpc-service;swift-remote-mirror;swift-remote-mirror-headers
-llvm-install-components=llvm-cov;llvm-profdata
+llvm-install-components=llvm-cov;llvm-profdata;install-IndexStore
```
Comment (compnerd) [inline utils/build-presets.ini]: Shouldn't that be `IndexStore` and not `install-IndexStore`?

**Comment by gottesmm** [comment]
@akyrtzi Are you sure that install-IndexStore is right? It should be just IndexStore.

**Comment by gottesmm** [comment]
@compnerd lol?! Who was first? Not sure. = p.

File: utils/build-presets.ini (hunk 1 above)
Comment (shahmishal) [reply utils/build-presets.ini]: Good catch!

Comment (akyrtzi) [reply utils/build-presets.ini]: Oooops, my bad.

**Comment by shahmishal** [comment]
**macOS Toolchain**
[Download Toolchain](https://ci.swift.org/job/swift-PR-toolchain-osx/100//artifact/branch-master/swift-PR-20132-100-osx.tar.gz)
Git Sha - 77c92ffc68b47f1a144b055e321229f4810ea551

**Install command**
`tar -zxf  swift-PR-20132-100-osx.tar.gz --directory ~/`

**Comment by shahmishal** [comment]
**Linux Toolchain** (Ubuntu 16.04)
[Download Toolchain](https://ci.swift.org/job/swift-PR-toolchain-Linux/83//artifact/branch-master/swift-PR-20132-83-ubuntu16.04.tar.gz)
Git Sha - 77c92ffc68b47f1a144b055e321229f4810ea551

**Install command**
`tar zxf  swift-PR-20132-83-ubuntu16.04.tar.gz`
[More info](https://swift.org/download/#using-downloads)

**Comment by shahmishal** [comment]
@akyrtzi Not sure, why GitHub did not update the status on the PR.

**Comment by akyrtzi** [comment]
Toolchains look good.

**Pull request merged**